### PR TITLE
Release

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +0,0 @@
-# doc-resources
-Ontotext documentation resources
-
-Adding this repository as submodule to another repository
```
Comment (stefan-enev) [inline README.md:4]: Why did you delete the readme?

Comment (renetapopova) [inline README.md:4]: I don't remember. Doino deleted it. See the history.
